nextcloudappstore/core/views.py

```diff
@@ -1,3 +1,7 @@
 from django.shortcuts import render
+from django.views.generic import View
 
-# Create your views here.
+
+class HomeView(View):
```

Comment:
Could probably be even shorter by subclassing TemplateView https://docs.djangoproject.com/en/1.9/ref/class-based-views/base/#templateview